### PR TITLE
Exposing methods to grab individual test results

### PR DESCRIPTION
**Pull request by bkern**
Exposing a method for getting all the individual unit testing results.

### CODE REVIEW

File: morphir/runtime/src/org/finos/morphir/runtime/TestSummary.scala [hunk 1]
```diff
@@ -9,7 +9,8 @@ private[runtime] case class CoverageInfo(
 case class TestSummary(
     report: String,
     countsByModule: Map[(PackageName, ModuleName), TestResultCounts],
-    private val coverage: CoverageInfo
+    private val coverage: CoverageInfo,
+    private val individualTestResults: Map[(PackageName, ModuleName), Map[String, String]]
```
Comment (edwardpeters) [inline morphir/runtime/src/org/finos/morphir/runtime/TestSummary.scala:13]: I think this should just be a map from FQName to some type which gives the test result - probably more than string.  SingleTestResult, or something?

File: morphir/runtime/src/org/finos/morphir/runtime/quick/testing/TestTree.scala [hunk 2]
```diff
@@ -251,9 +251,42 @@ private[runtime] object TestSet {
     TestSummary(
       toReport(testSet),
       testSet.modules.map(module => (module.pkgName, module.modName) -> ModuleTests.getCounts(module)).toMap,
-      coverageInfo
+      coverageInfo,
+      extractTestResults(testSet.modules)
     )
 
+  def extractTestResults(testSet: List[ModuleTests[SingleTestResult]])
```
Comment (edwardpeters) [inline morphir/runtime/src/org/finos/morphir/runtime/quick/testing/TestTree.scala:258]: I don't think we should be exposing `ModuleTests` - that's really an implementation detail of how the tests are generated.

File: morphir/runtime/src/org/finos/morphir/runtime/TestSummary.scala [hunk 3]
```diff
@@ -9,7 +9,8 @@ private[runtime] case class CoverageInfo(
 case class TestSummary(
     report: String,
     countsByModule: Map[(PackageName, ModuleName), TestResultCounts],
-    private val coverage: CoverageInfo
+    private val coverage: CoverageInfo,
+    private val individualTestResults: Map[FQName, String]
```
Comment (edwardpeters) [inline morphir/runtime/src/org/finos/morphir/runtime/TestSummary.scala:13]: This is better, but I don't like it just being to `String` - that makes it difficult to tell what the results were in a programatic way. Why can't this be `SingleTestResult`?

File: morphir/runtime/test/jvm/src/org/finos/morphir/runtime/UnitTestingSpec.scala [hunk 4]
```diff
@@ -57,7 +57,7 @@ object UnitTestingSpec extends MorphirBaseSpec {
     suite("Failing Project")(
       test("Show Results (for human readability check - ignore)") {
         getTestSummary.map { result =>
-          println(result)
+          // println(result)
```
[the remark this answers is not in the document]
Comment (bkern) [reply morphir/runtime/test/jvm/src/org/finos/morphir/runtime/UnitTestingSpec.scala]: removed thanks for catching that

**Review by edwardpeters — APPROVED** [review]
LGTM!

**Review by michelchan — APPROVED** [review]

**Pull request merged**